packages/sdk-react/src/components/onboarding/OnboardingPersonDocuments/OnboardingPersonDocuments.tsx

```diff
@@ -57,7 +57,7 @@ export const OnboardingPersonDocuments = () => {
 
     if (!fields) return undefined;
 
-    const { id, ...documents } = fields;
```

Comment:
I don't remember why I added this code.

But in this case, it is better to return `fields ` directly.

Reply:
Not sure, I understand suggestion. `id` was not used, that's why I removed it

<img width="871" alt="Screenshot 2024-07-31 at 21 43 57" src="https://github.com/user-attachments/assets/4442804a-0888-47b5-8550-e5a5552b4ed4">